.github/workflows/check-package.yml

```diff
@@ -3,15 +3,28 @@ name: Check package flow
 on:
```

Comment:
I did not review this

Reply:
that is fine it was check by @akihironitta and all need to pass :)